_posts/2017-04-06-git-aliases-ideas-from-our-setups.md

```diff
@@ -0,0 +1,72 @@
+---
+date: 2017-04-06
+title: "Git aliases: Ideas from our setups"
+categories:
+  - CLI
+  - Git
+  - Alias
+author_staff_member: 02_fausto
+medium_link: https://blabla.com
+---
+
+In this post we will keep an updated list of the Git aliases we use the most, particularly the ones we find interesting (i.e., not `c = commit`) and that have increased our productivity using Git.
```

Comment:
I don't think a post should change. I think we should make a series of posts and have a static page where we collect them all. What do you think?